src/install.rs

```diff
@@ -502,7 +502,19 @@ fn stash_saved_partitions(disk: &mut File, saved: &SavedPartitions) -> Result<()
 fn is_dasd(config: &InstallConfig) -> Result<bool> {
```

Comment:
WDYT about renaming this function to `is_eckd_dasd`?

Reply:
i don't have a strong opinion on that. it's a first patch in FBA-story, so once i figure out how to configure&use FBA, i would modify this function again